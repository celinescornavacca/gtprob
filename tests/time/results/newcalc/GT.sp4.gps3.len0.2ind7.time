0.38user 0.13system 0:00.73elapsed 70%CPU (0avgtext+0avgdata 43088maxresident)k
133inputs+0outputs (0major+16659minor)pagefaults 0swaps
