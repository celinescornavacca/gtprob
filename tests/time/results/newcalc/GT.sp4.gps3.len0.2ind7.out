-14.6487608126
